Major wiki restructure, minor fixes, rewording (#1616)
hyprwm/hyprland-wiki: content/configuring/core/autostart.md

---
weight: 90
title: Autostart
---

> Autostarting apps can be done by executing things on the `hyprland.start` event:

Reviewer: ```suggestion
Autostarting apps can be done using the `hyprland.start` event:
```
